platformio.ini

```diff
@@ -1482,12 +1482,13 @@ build_flags       = ${stm32_flash_drive.build_flags}
 # Espressif ESP32
 #
 [env:esp32]
-platform      = espressif32@1.11.2
+platform      = espressif32@2.1.0
 board         = esp32dev
 build_flags   = ${common.build_flags} -DCORE_DEBUG_LEVEL=0
 src_filter    = ${common.default_src_filter} +<src/HAL/ESP32>
 lib_ignore    = NativeEthernet
-upload_speed  = 115200
+upload_speed  = 500000
+monitor_speed = 250000
```

Comment:
Any ESP configuration examples that have the lower baud rate should be updated to this higher one. Or, this should be left at 115200 if the configurations are better to retain the "more reliable" lower baud rate.

Reply:
I recommend to leave it at 115200 for more stable and reliable operation. Higher baud rates may or may not work on some boards depending on which chip they are using for the USB interface.

Reply:
Hmm, looks like no examples are set up, so… off we go.

Reply:
Well, pio.ini can be excluded then, that's why i did separate commit.